### PR TITLE
Update description to shorter one and mention kubernetes

### PR DESCRIPTION
**Pull request by dgolovin**
This PR related to #1781.

Signed-off-by: Denis Golovin dgolovin@redhat.com

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "vscode-openshift-connector",
   "displayName": "OpenShift Connector",
-  "description": "Interacting with Red Hat OpenShift clusters and providing a streamlined developer experience using Visual Studio Code",
+  "description": "Simplified app dev for Kubernetes or Red Hat OpenShift",
```
Comment (mohitsuman) [inline package.json:4]:
```suggestion
  "description": "Simplified application development with Kubernetes and Red Hat OpenShift",
```

**Review by mohitsuman — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (dgolovin) [reply package.json:4]: Why you did not tell anything in the issue till now?

Comment (dgolovin) [reply package.json:4]: I am pushing option 7, for the reasons I noted in the issue.

Comment (mohitsuman) [reply package.json:4]: I will discuss in the issue, but I do not think short forms justify the workflow of the extension.

**Pull request merged**